src/layouts/MainLayout/MainLayout.js

```diff
@@ -3,16 +3,18 @@ import PropTypes from 'prop-types';
 
 import Navbar from 'components/Navbar';
 
-import styles from './MainLayout.module.scss';
+import { MainContainer, MainContainerWithoutHeader } from './MainLayout.styles';
 
-const MainLayout = ({ children, noHeader }) => (
-  <>
-    {!noHeader && <Navbar />}
-    <main className={noHeader ? styles.noHeader : styles.mainContainer}>
-      {children}
-    </main>
-  </>
-);
+const MainLayout = ({ children, noHeader }) => {
+  const Component = noHeader ? MainContainerWithoutHeader : MainContainer;
+
+  return (
+    <>
+      {!noHeader && <Navbar />}
+      <Component>{children}</Component>
```

Comment:
why don't you just define a `MainContainer` styled component that receives a prop called "withHeader" and set the styles conditionally in the styles file? that's one of the biggest advantages of css-in-js, to extract all the styling logic to it